include/z64environment.h

```diff
@@ -132,17 +132,36 @@ typedef struct {
     /* 0x06 */ u8 skybox2Index;
 } TimeBasedSkyboxEntry; // size = 0x8
 
+// Get blend rate from `EnvLightSettings.blendRateAndFogNear` in 0-255 range
+#define ENV_LIGHT_SETTINGS_BLEND_RATE_U8(blendRateAndFogNear) (((blendRateAndFogNear) >> 10) * 4)
+#define ENV_LIGHT_SETTINGS_FOG_NEAR(blendRateAndFogNear) ((blendRateAndFogNear) & 0x3FF)
+
 typedef struct {
     /* 0x00 */ u8 ambientColor[3];
     /* 0x03 */ s8 light1Dir[3];
     /* 0x06 */ u8 light1Color[3];
     /* 0x09 */ s8 light2Dir[3];
     /* 0x0C */ u8 light2Color[3];
     /* 0x0F */ u8 fogColor[3];
-    /* 0x12 */ s16 fogNear;
-    /* 0x14 */ s16 fogFar;
+    /* 0x12 */ s16 blendRateAndFogNear;
+    /* 0x14 */ s16 zFar;
```

Comment:
I think we should more explicitly document the nuance that fog far is always going to be max, so the same as zFar so changing one is effectively changing the other

I say to add a brief description here (or in the live version of the struct i guess?), cause if I was new to this codebase id be confused by "near and far" being next to each other but they are for different things

Reply:
The effective values end up in `LightContext`, where I have some comments already, so I'm not sure where to write anything now

Reply:
I think the environment context is the more user-facing system/struct and is where people would go to read about this first